.github/workflows/ci.yml

```diff
@@ -26,3 +26,102 @@ jobs:
         - php: 8.2
           phpunit: true
           phpunit_suite: recipe-cms
+        - php: 8.3
+          phpunit: true
+          phpunit_suite: recipe-cms
```

Comment:
```suggestion
```

Reply:
Seems like we should keep it here - we support [php 8.3 in 5.2](https://www.silverstripe.org/blog/cms-5-2/) - we currently have an 8.3 test in the `5` [branch of sink](https://github.com/silverstripe/recipe-kitchen-sink/blob/5/.github/workflows/ci.yml#L29), so may as well backport that to 5.2 as well

Reply:
Ahh, I thought 8.3 was in 5.3. This is fine then.